### PR TITLE
enable pyensight api coverage

### PR DESCRIPTION
**Pull request by mariostieriansys**
With the previous change the coverage of the pyensight API got disabled.
The PR fixes it.

PyEnSight, compared to the other projects, builds the 95% of its code in the API repo. This is because PyEnSight replicates already existing Python APIs in EnSight via introspection.

In the previous PRs it was decided to let anything that is automated (in this case, pure Python scripts) in the API repo. However, since the API repo, as mentioned, does not generate only grpc bindings but also actual code, it is important that this is reported in the code coverage

### CODE REVIEW

**Review by randallfrank — DISMISSED** [review]

**Comment by mariostieriansys** [comment]
@jorgepiloto how can I change the baseline for CodeCov?

Because it was missing the definitions of the coverage from ansys.api.pyensight, CodeCov still reports that the coverage has not changed, but that's not the case

**Comment by jorgepiloto** [comment]
We can try to remove all data in Codecov and just trigger it again using the CI. This should fix the issue. It will only loose coverage history from this pull-request backwards.

**Comment by mariostieriansys** [comment]
@jorgepiloto that would be great

**Comment by jorgepiloto** [comment]
It's done. Let's give it a second try.

**Comment by mariostieriansys** [comment]
@jorgepiloto still 64%. With the coverage of the API it should bump to 98%. I believe there's something we need to change so that it takes not only the src folder for coverage but also ansys/api/pyensight

**Comment by mariostieriansys** [comment]
On monday I will look at the codecov action parameters and see if we can do something for it

File: doc/source/_static/ensight_rest_v1.yaml [hunk 1]
```diff
@@ -0,0 +1,594 @@
+openapi: 3.0.0
```
Comment (randallfrank) [inline doc/source/_static/ensight_rest_v1.yaml:1]: I'm not sure we want two copies of the openapi file.  Should this not be a copy command in the yaml script?  Just before building the docs, copy the file over into the static directory?

Comment (mariostieriansys) [reply doc/source/_static/ensight_rest_v1.yaml:1]: @randallfrank the problem is that when I cleaned the old "dev" and "release" folders in gh-pages, this was the only one with a broken link. I have placed manually the file on the new gh-pages folders, but copying it should work as well. I am looking at the codecov issue right now but I can change it for sure

**Comment by mariostieriansys** [comment]
@jorgepiloto @randallfrank 
I have looked for documentation on both the codecov action and the codecov website on covering two repositories simultaneously with one upload.
I haven't found a solution.

So, I believe we can start merging this, that at least enables the coverage of the api repository during the pytests (if we are interested one can look at the logs there), while after this has been merged I'll work on increasing the coverage of the pyensight repo itself. It was higher than 80% when I created the first batch of tests, but we have added a few files. It shouldn't take me too much to increase it. But I need this to go in first

File: doc/source/_static/ensight_rest_v1.yaml (hunk 1 above)
Comment (mariostieriansys) [reply doc/source/_static/ensight_rest_v1.yaml]: I believe I added the copy step

**Pull request closed without merging**